backend/app.ts

```diff
@@ -0,0 +1,41 @@
+import mongoose = require('mongoose');
+import express = require('express');
+import path = require('path');
+import bodyParser = require('body-parser');
+import cors = require('cors');
```

Comment:
What's `cors` used for?

Reply:
Cross-Origin Resource Sharing (CORS). Used for headers.